conda/cli/helpers.py

```diff
@@ -14,6 +15,11 @@
 )
 
 
+def show_help_on_empty_command():
+    if len(sys.argv) == 1:  # sys.argv == ['/path/to/bin/conda-env']
+        sys.argv.append("--help")
```

Comment:
Since this is so tiny and only used once can we inline it and mark the original for deprecation?